src/features/device-state/routes/state-patch-v3.ts

```diff
@@ -297,12 +285,25 @@ export const statePatchV3: RequestHandler = async (req, res) => {
 				}
 			}
 
-			if (apps != null || Object.keys(deviceBody).length > 0) {
+			if (
+				Object.keys(metricsBody).length > 0 &&
+				!(await shouldUpdateMetrics(uuid))
```

Comment:
There's a special case I think where we always want to recognize changes to the `is_undervolted` property which we currently always accept but this PR has moved to being treated like any of the other "metrics" fields